lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -1270,11 +1270,7 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
   def alert_ids(%__MODULE__{} = t), do: [t.alert.id]
 
   def valid_candidate?(%__MODULE__{} = t) do
-    suppressed =
-      t.alert.id in ["549108", "555079", "555080"] and
-        t.location_context.home_stop === "place-gover"
-
-    not suppressed
+    t.alert.id not in ["197140", "580015"]
```

Comment:
Where did these IDs come from? When I try to look up either of them in the API, I get a 404 error. Looking at the alerts currently in the feed, it seems like these should be `571855` and `578253`.

Reply:
`197140` is the alert ID that Dave created in test. I can put the values in variables so that they are more obviously labeled. The other ID is in prod so I'm not sure why the API isn't returning it. I found it on the `Future` tab in AlertsUI. Based on the discussion in [this thread](https://mbta.slack.com/archives/C03K6NLKKD1/p1718222171213539), the alert that will be used for the closure will be a station closure instead of a service change. That may be why `571855` starts and ends at the same time. They are only using it in notifications but will switch to `580015` when the alert is in effect. I will triple check with Dave.

Reply:
Huh, okay, I think I see what they're doing there... it doesn't really make sense to me (why two alerts? how is the second one being hidden from the feed?), but the ID is at least correct to what they've told us.

Incidentally, it looks like 571855 is now active _currently_, until tonight. Maybe we should add that one here in addition to 580015?

Reply:
Because `571855` is a Service Change, it won't show on these screens anyway.